packages/grid/x-data-grid/src/components/GridErrorOverlay.tsx

```diff
@@ -4,15 +4,14 @@ import { GridOverlay, GridOverlayProps } from './containers/GridOverlay';
 import { useGridSelector } from '../hooks/utils/useGridSelector';
 import { gridDensityRowHeightSelector } from '../hooks/features/density/densitySelector';
 
-export interface ErrorOverlayProps extends GridOverlayProps {
+export interface GridErrorOverlayProps extends GridOverlayProps {
   error?: Error;
   hasError: boolean;
   errorInfo: any;
 }
 
-// TODO v6: rename to GridErrorOverlay
-export const ErrorOverlay = React.forwardRef<HTMLDivElement, ErrorOverlayProps>(
-  function ErrorOverlay(props: ErrorOverlayProps, ref) {
+export const GridErrorOverlay = React.forwardRef<HTMLDivElement, GridErrorOverlayProps>(
+  function GridErrorOverlay(props, ref) {
```

Comment:
We should include the renamed exports in the migration guide

Reply:
This file was not exported previously, only being used in `slots`, may be export it then?
But as it will be a new export it won't go in migration guide, right?

Reply:
Oh, indeed.
No need to export it then

Reply:
Ok, thanks for the update!